src/components/Article.js

```diff
@@ -223,7 +235,8 @@ const ArticleInner = ({ lang, articleTitle, initialAnchor }) => {
     { name: 'sections', enabled: true, handler: showArticleTocPopup },
     { name: 'quickfacts', enabled: !!article.infobox, handler: showQuickFacts },
     { name: 'gallery', enabled: !!article.media.length, handler: showGallery },
-    { name: 'languages', enabled: article.languageCount, handler: showArticleLanguagePopup }
+    { name: 'languages', enabled: article.languageCount, handler: showArticleLanguagePopup },
+    { name: 'share', enabled: !!window.MozActivity, handler: shareArticleUrl }
```

Comment:
Similar to `shareArticleUrl()` we should propagate `shareEnabled` instead of repeating `!!window.MozActivity`